### ISSUE TITLE
Modification and roadmap information

### ISSUE THREAD

**Issue by ndcuong91** [titikid/c3d_luanvan#1]
### Modification list from original code:
   * [x]  Delete all .fc6, .fc7, .prob vector in result folder
   * [x]  Change result folder (to locate .fc6, .fc7, .prob vector) in **output/result**
   * [x]  Don't delete result folder (e.g. Kinect_1_test_on_Kinect_1), add time to it
   * [x]  Finetuning continuously for better experiment on params
   * [x]  Add arguments for more flexible when training
   * [x]  Add data_augmentor.py script
   * [x]  Add params.txt in output result folder which list all the parameters used for finetuning
   * [x]  Train & test set can use different data_type (e.g. train on K_1_original, test on K_1_segmented)
   * [x]  Can use average features for classification or only first feature (for first 16 frames)
   * [x]  Check GPU ready
   * [x]  Use center crop instead of random crop in training
   * [x]  Add summary result  
   * [x]  Modify net.cpp to run with different filter in convolution

### Road map (15/12/2018 --> 30/03/2019)
   * Modify code, Explore dataset, Preprocessing and Tuning hypeparameters
   *  [x]  Data augmentation  
   *  [x]  Modify input size for best result  
   *  [x]  Find the best way to pre-processing (pre 3)  
   *  [ ]  Find the most suitable hyper-parameters set 

   * Run code with 12 hand-gesture dataset
   *  [x]  Finish create dataset

   * Run code with 12 hand-gesture dataset
   * [ ]  Make simpler network 
   * [ ]  Change loss function
   * [ ]  Find the best way for training that can achieve good result on every view
